dev/scripts/src/alluxio.org/build-distribution/cmd/common.go

```diff
@@ -55,12 +55,6 @@ type module struct {
 	mavenArgs string // maven args for building the module
 }
 
-var fuseUfsModules = map[string]module{
```

Comment:
avoid copying the module part by keeping a variable listing the keys of the map that you want as fuse ufs modules

Reply:
Yeah, copy all the map fields do not make sense

Reply:
another way is to add a field to the module
all ufs modules - default ufs modules - core ufs modules
so that there is hierachy, but will discuss to see whether we actually need hadoop jars